README.md

```diff
@@ -87,7 +88,11 @@ add_subdirectory(${memilio_SOURCE_DIR}/cpp ${memilio_BINARY_DIR})
 
 Finally, create compilation targets for the `.cpp`-files, and link all required libraries, like memilio or the model libraries used by the simulation.
 
-- In the global `CMakeLists.txt`, add an option `BUILD_<Year>_<FirstAuthor>_et_al_<FirstWordTitle>` for your new content and the commands to build your files using the local `CMakeLists.txt`:
+- In the global `CMakeLists.txt`, add an option `BUILD_<Year>_<FirstAuthor>_et_al_<FirstWordTitle>` for your new content and set the default option to `OFF`:
+```bash
+option(BUILD_<Year>_<FirstAuthor>_et_al_<FirstWordTitle> "Build simulations from folder <FolderName>." OFF)
+```
+- Additionally, add the commands to build your files using the local `CMakeLists.txt`: 
```

Comment:
```suggestion
Additionally, add the commands to build your files using the local `CMakeLists.txt`: 
```

Why did you add this bullet? I would remove it to make clear that this is still related to the global CmakeLists. As an alternative, state that this is again related to the gloabl CmakeLists..